website/components/io-home-card/index.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react'
+import Link from 'next/link'
+import InlineSvg from '@hashicorp/react-inline-svg'
+import camelCase from 'camelcase'
+import classNames from 'classnames'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+import { productLogos } from './product-logos'
+import s from './style.module.css'
+
+interface IoHomeCardProps {
+  variant?: 'light' | 'gray' | 'dark'
+  products?: Array<keyof typeof productLogos>
+  link: {
+    url: string
+    type: 'inbound' | 'outbound'
+  }
+  inset?: 'none' | 'sm' | 'md'
+  eyebrow?: string
+  heading?: string
+  description?: string
+  children?: React.ReactNode
+}
+
+function IoHomeCard({
+  variant = 'light',
+  products,
+  link,
+  inset = 'md',
+  eyebrow,
+  heading,
+  description,
+  children,
+}: IoHomeCardProps) {
+  const space = camelCase(['space', inset])
```

Comment:
Alternate idea I had, potentially use another localized `--space-${inset}` CSS var so a package isn't needed.

Let me know if this suggestion is not clear!

Reply:
https://github.com/hashicorp/vault/pull/13151/commits/05138f348d33eb7a219b42603af37dc9af816615